system/libarc32_arduino101/drivers/ns16550.c

```diff
@@ -340,7 +340,7 @@ unsigned char uart_poll_out(
 	)
 {
 	/* wait for transmitter to ready to accept a character */
-	while ((INBYTE(LSR(which)) & LSR_TEMT) == 0)
```

Comment:
Why is TEMT not sufficient? With our UART configuration it should be set when both THR and FIFO are empty, right?